geomesa-accumulo/geomesa-accumulo-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapreduce/GeoMesaAccumuloInputFormat.scala

```diff
@@ -173,60 +171,29 @@ class GeoMesaAccumuloInputFormat extends InputFormat[Text, SimpleFeature] with L
     accumuloSplits
   }
 
-  override def createRecordReader(split: InputSplit, context: TaskAttemptContext) = {
+  override def createRecordReader(split: InputSplit, context: TaskAttemptContext): GeoMesaRecordReader[AccumuloFeatureIndex] = {
```

Comment:
Actually, I take that back.  The file changed considerably so I'm going to leave this auto-fix in place.